### ISSUE TITLE
Error reading transaction.ref_block_num when using newaccount on 6.1.4

### ISSUE THREAD

**Issue by nsjames** [EOSIO/eosjs#22]
Node version: v8.9.4
NPM version: 5.6.0
OS: Win7
EOS OS: Ubuntu 16.10 ( VM )

Something seems to have changed since version `6.1.0` regarding account creation.
Though I'm not sure if it's with eosjs itself, but perhaps a package it relies on as reverting to `6.1.0` after updating does not fix the issue. ( I had an older repo that worked, then I updated and no longer works ).

Other chain methods seem to work fine including transactions. In fact it doesn't seem to be hitting the EOS node at all when doing `newaccount`

Code used to produce
```
let eos = Eos.Localnet({httpEndpoint:`http://${network.host}:${network.port}`, keyProvider:'5KQwrPbwdL6PhXujxW37FSSQZ1JiwsST4cqQzDeyXtP79zkvFD3'});
eos.newaccount({
	creator: 'inita',
	name: 'tester',
	owner: 'EOS7aewJRtiZfSfdzKZyuDMQEhreNXPgZsqMTmhZFGFqP3aXHZaFi',
	active: 'EOS7aewJRtiZfSfdzKZyuDMQEhreNXPgZsqMTmhZFGFqP3aXHZaFi',
	recovery: 'inita',
	deposit: `1 EOS`
}).then(acc => {
	console.log(acc);
})
```

Error:

```
struct.js:79 Error reading transaction.ref_block_num in data:
fromByteBuffer @ struct.js:79
fromBuffer @ fcbuffer.js:276
(anonymous) @ write-api.js:520
(anonymous) @ write-api.js:660
(anonymous) @ exported-helpers.js:65
(anonymous) @ exported-helpers.js:75
callback @ process-args.js:70
callback @ apigen.js:85
(anonymous) @ apigen.js:109
Promise resolved (async)
(anonymous) @ apigen.js:104
(anonymous) @ exported-helpers.js:54
(anonymous) @ exported-helpers.js:75
callback @ process-args.js:70
callback @ apigen.js:85
(anonymous) @ apigen.js:109
Promise resolved (async)
(anonymous) @ apigen.js:104
createTransaction @ exported-helpers.js:48
api.(anonymous function) @ apigen.js:38
transaction @ write-api.js:496
(anonymous) @ write-api.js:179
(anonymous) @ write-api.js:320
document.getElementById.addEventListener @ index-test.ts:39
bytebuffer.js:2795 ByteBufferAB(offset=2,markedOffset=-1,limit=194,capacity=194)
-------------------------------------------------------------------
 97 04<3E DA D9 86 B0 90 7F 5A 01 00 00 00 00 00   ..>......Z......
 93 DD 74 00 01 00 00 00 00 00 00 30 55 00 40 9E   ..t........0U.@.
 9A 22 64 B8 9A 01 00 00 00 00 00 93 DD 74 00 00   ."d..........t..
 00 00 A8 ED 32 32 8A 01 00 00 00 00 00 93 DD 74   ....22.........t
 00 00 00 40 5C 95 B1 CA 01 00 00 00 01 03 62 97   ...@\.........b.
 9E 69 82 84 0E 3A D5 A3 11 C0 8D 04 6A A9 18 03   .i...:......j...
 81 A6 60 10 18 C6 80 45 B7 C6 01 B5 5E 23 01 00   ..`....E....^#..
 00 01 00 00 00 01 03 62 97 9E 69 82 84 0E 3A D5   .......b..i...:.
 A3 11 C0 8D 04 6A A9 18 03 81 A6 60 10 18 C6 80   .....j.....`....
 45 B7 C6 01 B5 5E 23 01 00 00 01 00 00 00 00 01   E....^#.........
 00 00 00 00 00 93 DD 74 00 00 00 00 A8 ED 32 32   .......t......22
 01 00 10 27 00 00 00 00 00 00 04 45 4F 53 00 00   ...'.......EOS..
 00 00>                                            ..

apigen.js:111 Error: Assertion failed (transaction.ref_block_num) transaction.ref_block_num
    at assert (bn.js:6)
    at BN._initArray (bn.js:145)
    at BN.init [as _init] (bn.js:82)
    at new BN (bn.js:39)
    at Object.fromByteBuffer (types.js:476)
    at Object.fromByteBuffer (struct.js:66)
    at Function.fromBuffer (fcbuffer.js:276)
    at write-api.js:520
    at write-api.js:660
    at exported-helpers.js:65
```

**Comment by nsjames** [comment]
Actually, this is happening with all transactions. 
Non transaction methods seem to work though ( getBlock, getInfo, etc )

**Comment by nsjames** [comment]
A few things I noticed after trying to debug the issue on the eosjs repo itself.

I cloned the latest eosjs and wrote an integration test within the repository using the same code from the readme and it works fine within mocha. I then wrote a test within the the repo that I'm working on with the exact same procedure and it again works fine.

It seems to only be a problem when trying to use eosjs from within the browser itself.

**Comment by jcalfee** [comment]
This is a partial fix, unless you try to use a type that is greater than 64 bits .. The exchange contract has a type like this.  For some reason I did something with BN.js that is not working under browserify and as you reported webpack. 

published version 6.1.5

**Comment by nsjames** [comment]
Thanks @jcalfee !

Very weird indeed that it doesn't work when bundled with browserify/webpack. 
It's possible it has something to do with one of these:
https://github.com/indutny/bn.js/pull/168
https://github.com/indutny/bn.js/issues/160
which possibly insinuates that `Buffer` might be being overwritten when bundled, but when we use it in the test suite it's the only one present.
